### DISCUSSION TITLE
Get the neighborhood of a node

### DISCUSSION THREAD

**Discussion by manuel-dileo** [pyg-team/pytorch_geometric#7090] [Q&A] [answered]
Is there an easy way to get the neighborhood of a node from the edge_index on a graph with millions of nodes?

**Comment by manuel-dileo** [comment]
Ok, edge_index[1, edge_index[0] == node_idx] works well (I found this line of code in an answer on common neighbors). Maybe adding a method "neighbors" in torch_geometric.utils could be useful to find the solution to this problem fast? I can open a pull request on it

**Comment by EdisonLeeeee** [comment, marked as answer]
You can also take a look at [torch_geometric.utils.k_hop_subgraph](https://pytorch-geometric.readthedocs.io/en/latest/modules/utils.html#torch_geometric.utils.k_hop_subgraph)

**Reply by manuel-dileo** [reply]
Oh ofc, thank you!
